core/modules/system/system.admin.inc

```diff
@@ -1000,16 +999,21 @@ function system_modules_submit($form, &$form_state) {
   }
 
   // Check if a previous search is present and preserve it on redirect.
-  if (!empty($form_state['values']['search'])) {
+  if (!empty($form_state['values']['search']) && sizeof($actions['enable']) != 0)) {
```

Comment:
this is fine as-is as we probably don't need the search term if the module is being uninstalled...

Reply:
Doo you mean the original is fine or my change is fine?